stack.yml

```diff
@@ -7,7 +7,7 @@ services:
     environment:
     - DB_SETUP=automatic
     - DB_SETUP_PASS=8Ex4zOj5Y44g-axlXoF_s
-    - SIMPLERISK_DB_HOSTNAME=mysql
+    - SIMPLERISK_DB_HOSTNAME=mariadb
```

Comment:
This file is generated automatically (`update_stack_and_workflows.sh`), so no matter what changes are done here, this will be overwritten.

Reply:
Also, the stack uses MySQL, so although this change fixes the issue, the problem is that it does not reflect the DB being used, so the change should be on the DB container definition (line 16) instead.